release/tune_tests/scalability_tests/run.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+ray_version="" 
+commit=""
+ray_branch=""
+
+for i in "$@"
+do
+echo "$i"
+case "$i" in
+    --ray-version=*)
+    ray_version="${i#*=}"
+
+    ;;
+    --commit=*)
+    commit="${i#*=}"
+    ;;
+    --ray-branch=*)
+    ray_branch="${i#*=}"
+    ;;
+    --workload=*)
+    workload="${i#*=}"
+    ;;
+    --help)
+    usage
+    exit
+    ;;
+    *)
+    echo "unknown arg, $i"
+    exit 1
+    ;;
+esac
+done
+
+if [[ $ray_version == "" || $commit == "" || $ray_branch == "" ]]
+then
+    echo "Provide --ray-version, --commit, and --ray-branch"
+    exit 1
+fi
+
+echo "version: $ray_version"
+echo "commit: $commit"
+echo "branch: $ray_branch"
+echo "workload: ignored"
+
+wheel="https://s3-us-west-2.amazonaws.com/ray-wheels/$ray_branch/$commit/ray-$ray_version-cp37-cp37m-manylinux2014_x86_64.whl"
+
+pip install -U pip
+pip install -U "$wheel"
+pip install "ray[tune]" "ray"
+pip install boto3==1.4.8 cython==0.29.0
+
+if ! python "wait_cluster.py" 16 450; then
```

Comment:
i wish we didn't have to implement this script... but ok looks fine for now

Reply:
Yes, this seems like a useful utility for the release testing tool (or maybe cluster management)